x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/form/validations.ts

```diff
@@ -272,3 +273,103 @@ export const minAgeGreaterThanPreviousPhase =
       }
     }
   };
+
+export const downsampleIntervalMultipleOfPreviousOne =
+  (phase: PhaseWithDownsample) =>
+  ({ formData }: { formData: Record<string, any> }) => {
+    if (phase === 'hot') return;
+
+    const getValueFor = (_phase: PhaseWithDownsample) => {
+      const intervalSize = formData[`_meta.${_phase}.downsample.fixedIntervalSize`];
+      const intervalUnits = formData[`_meta.${_phase}.downsample.fixedIntervalUnits`];
+
+      if (!intervalSize || !intervalUnits) {
+        return null;
+      }
+
+      const milliseconds = moment.duration(intervalSize, intervalUnits).asMilliseconds();
+      const esFormat = intervalSize + intervalUnits;
+
+      return {
+        milliseconds,
+        esFormat,
+      };
+    };
+
+    const intervalValues = {
+      hot: getValueFor('hot'),
+      warm: getValueFor('warm'),
+      cold: getValueFor('cold'),
+    };
+
+    const checkIfGreaterAndMultiple = (nextInterval: number, previousInterval: number): boolean =>
+      nextInterval > previousInterval && nextInterval % previousInterval === 0;
+
+    if (phase === 'warm' && intervalValues.warm) {
+      if (intervalValues.hot) {
+        if (
+          !checkIfGreaterAndMultiple(
+            intervalValues.warm.milliseconds,
+            intervalValues.hot.milliseconds
+          )
+        ) {
+          return {
+            message: i18n.translate(
+              'xpack.indexLifecycleMgmt.editPolicy.downsamplePreviousIntervalWarmPhaseError',
+              {
+                defaultMessage:
+                  'Must be greater than and a multiple of the hot phase value ({value})',
+                values: {
+                  value: intervalValues.hot.esFormat,
+                },
+              }
+            ),
+          };
```

Comment:
Do we want to provide a pre-filled number for downsampling here? One thing that is tricky about this form is how big it is and I feel like scrolling between phases you can easily get distracted by all the shiny things.

One alternative I thought of was providing a list of multiples in the error message like: (ex. 10 days, 20 days...)

Might not be worth the effort.

Reply:
> Do we want to provide a pre-filled number for downsampling here?

As I understand, this isn't a low effort because this wouldn't be just a static value and have to depend on the previous step: 
e.g., if someone configured a 1-day interval in a hot phase and then enabled a warm phase, then we should prefill with e.g., 2 days. As I understand, such support for "dynamic" initial values needs to be implemented first

Also really not sure what could be good defaults here. 

> One alternative I thought of was providing a list of multiples in the error message like: (ex. 10 days, 20 days...)

Could be!

Reply:
lets worry about optimizations and better defaults in subsequent PRs. hard to say what the rule should be without spending some time with it / hearing from users